src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Pages/Components/Surfaces/Card/BitCardDemo.razor.scss

```
@@ -1 +1,39 @@
+// The stand-in for a real picture in the Cover template example.
+.example-cover {
+    color: white;
+    display: flex;
+    height: 7rem;
+    font-size: 1.25rem;
+    align-items: center;
+    justify-content: center;
+    background: linear-gradient(120deg, var(--bit-clr-pri), var(--bit-clr-inf));
+}
+
+::deep {
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)